src/openloco/windows/CompanyWindow.cpp

```diff
@@ -422,9 +422,71 @@ namespace openloco::ui::windows::CompanyWindow
             self->call_viewport_rotate();
         }
 
+        static void viewport_centre_on_tile(loc16 loc, window* self)
+        {
+            registers regs;
+            regs.ax = loc.x;
+            regs.cx = loc.y;
+            regs.dx = loc.z;
+            regs.esi = (int32_t)self;
+            call(0x004C6827, regs);
```

Comment:
This function is also called by the town window, and possibly others. Probably best to move it into the viewport manager?